spec/support/request_helpers.rb

```diff
@@ -0,0 +1,7 @@
+module Request
+  module JsonHelpers
+    def json_response
+      @json_response ||= JSON.parse(response.body, symbolize_names: true)
```

Comment:
This seems kind of magical. I would remove this module and just call `JSON.parse` in the test.

Reply:
This helper module not only removes duplication in current tests but also (I imagine future tests will be using it as well) hides the detail that is not cared for in the tests.

Reply:
The problem is that using a module makes it very hard to know where that method came from. In fact, until looking at this file, I had no idea that `json_response` was something you defined.

I am fine with extracting the duplication into a function, however it should be explicit as to where the function comes from. 

https://8thlight.com/blog/eric-meyer/2012/11/16/composition-over-mixins.html

Reply:
ok. removing it and sticking with JSON.parse.